recipes/_configure_server.rb

```diff
@@ -45,7 +50,9 @@
 
     server_stanzas[stanza] = {}
     server_stanzas[stanza]['master_uri'] = master_uri
-    server_stanzas[stanza]['pass4SymmKey'] = pass unless pass.empty?
+
```

Comment:
Nit: you added extra ![spaaaaaaaaaaccccceeee](http://images5.fanpop.com/image/photos/29100000/Spaaace-space-core-29126462-225-225.jpg) (and line 55)